Makefile

```diff
@@ -67,14 +67,14 @@ debug:
 # for our stuff, kills optimization, and redirects to dev null so we
 # see any build failures.
 	make clean > /dev/null
-	CFLAGS='-g -O0' python3 -m pip install --global-option="build_ext" . > /dev/null
+	CFLAGS='-g -O0' python3 -m pip -v install --global-option="build_ext" . > /dev/null
 
 .PHONY: release-test
 release-test:
-	python3 -m pip install -e .[tests]
```

Comment:
This is a release time smoke test. It's up to the release managers to say if they want the verbose here.

Reply:
I've never needed to pay attention in detail to the output of this. I don't need verbose, but don't oppose it if anyone wants it. (It might push something useful off the top of the logs buffer.)